cmd/botkube/main.go

```diff
@@ -67,10 +67,15 @@ func main() {
 
 // run wraps the main logic of the app to be able to properly clean up resources via deferred calls.
 func run() error {
+	// Set up context
```

Comment:
If that's OK with you, I would like to have the `run(ctx)` and those 3 commands moved to `main` 🙂

Reply:
That works for me 👍